operations/_scripts/generate/generate_tf_vars.sh

```diff
@@ -22,74 +14,266 @@ else
 fi
 
 GITHUB_IDENTIFIER="$($GITHUB_ACTION_PATH/operations/_scripts/generate/generate_identifier.sh)"
+GITHUB_IDENTIFIER=$(echo $GITHUB_IDENTIFIER | tr "_" "-")
```

Comment:
should this be moved into the `generate_identifier.sh` script?

Probably create a ticket as I think this change would be a medium effort at least

Reply:
Originally I tried that and yes there are certainly some changes required to have that work. 
Ticket incoming

Reply:
https://github.com/bitovi/github-actions-deploy-docker-to-ec2/issues/105